charts/pega/templates/_pega-eks-ingress.tpl

```diff
@@ -7,7 +7,7 @@ metadata:
   namespace: {{ .root.Release.Namespace }}
   annotations:
     # Ingress class used is 'alb'
-    kubernetes.io/ingress.class: alb
+    kubernetes.io/ingress.class: {{ include "ingressClass" . }}
```

Comment:
Looks like, this template is specific to EKS, Can we hard code the ingress.class ```alb```, instead of deriving?